app_dart/bin/server.dart

```diff
@@ -78,11 +78,6 @@ Future<void> main() async {
       '/api/push-gold-status-to-github': PushGoldStatusToGithub(config, authProvider),
       '/api/push-engine-build-status-to-github': PushEngineStatusToGithub(config, authProvider, luciBuildService),
       '/api/refresh-chromebot-status': RefreshChromebotStatus(config, authProvider, luciBuildService),
-      '/api/refresh-github-commits': RefreshGithubCommits(
```

Comment:
The use case is a bad Cocoon roll out or GitHub is down. Otherwise, the new scheduling logic has a higher consistency rate than the cron method (previously dropped ~1% of commits now 0%).